media/css/mozorg/leadership.less

```diff
@@ -0,0 +1,901 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import (reference) '../pebbles/lib.less';
+@import '../pebbles/components/masthead.less';
+@import '../pebbles/components/footer.less';
+
+#masthead {
+    background: @color-mozred;
+    color: #fff;
+
+    .toggle {
+        background-image: url('/media/img/pebbles/icon-menu-light.svg');
+    }
+
+    @media @mq-tablet {
+        a {
+            color: #fff;
+        }
+    }
+}
+
+// Header
+.snapshot-header {
+    display: none;
+    background: #e9dbac url('/media/img/sandstone/grain.png');
+    width: 100%;
+    overflow: hidden;
+    position: relative;
+
+    @media @mq-tablet {
+        display: block;
+        height: 300px;
+    }
+
+    @media @mq-desktop {
+        height: 500px;
+    }
+}
+
+.snap {
+    display: inline-block;
+    background: #fff;
+    padding: 20px 20px 50px 20px;
+    box-shadow: 0 2px 2px 2px rgba(0, 0, 0, .2);
+    position: absolute;
+
+    span {
+        .background-size(230px, 230px);
+        background-color: #e8ebd3;
+        background-position: center center;
+        background-repeat: no-repeat;
+        display: block;
+        height: 230px;
+        outline: 1px solid rgba(0, 0, 0, .2);
+        width: 230px;
+    }
+
+    img {
+        display: block;
+        height: 230px;
+        margin: 0;
+        width: 230px;
+    }
+}
+
+.foreground {
+    display: none;
+
+    .snap img {
+        opacity: .1;
+        -webkit-animation: develop 2500ms ease-in forwards;
+                animation: develop 2500ms ease-in forwards;
+        -webkit-filter: invert(40%) sepia(80%);
+                filter: invert(40%) sepia(80%);
+    }
+
+    @media @mq-tablet {
+        display: block;
+        height: 300px;
+        left: 50%;
+        margin-left: -(@width-tablet / 2);
+        position: absolute;
+        top: 0;
+        width: @width-tablet;
+        z-index: 10;
+
+        .snap {
+            position: absolute;
+        }
+
+        .shot-one { // MSurman
+            left: -12px;
+            top: 6px;
+            transform: rotate(-3deg);
```

Comment:
Do we need to use the `.transform()` mixin here too, or are we not worried about slightly older webkit browsers?